android/app/src/main/java/io/mosip/registration_client/MainActivity.java

```diff
@@ -138,6 +180,7 @@ public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
                                 case "masterDataSync":
```

Comment:
We have moved to pigeon, any reason this method channel is present in the codebase

Reply:
@aviral-mishra-hunter  @Piyush7034